php/index.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$name = 'AnnieSenchenko';
+
+function rewrite($name){
+    $pattern = '#([A-Z][^A-Z]*)#';
```

Comment:
Using of unicode in regex: 
http://php.net/manual/en/reference.pcre.pattern.modifiers.php
http://php.net/manual/ru/regexp.reference.unicode.php